docs/src/snap/howto/storage/cloud.md

```diff
@@ -0,0 +1,476 @@
+  kind: Role
+  name: extension-apiserver-authentication-reader
+subjects:
+  - apiGroup: ""
+    kind: ServiceAccount
+    name: cloud-controller-manager
+    namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:cloud-controller-manager
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - create
+  - patch
+  - update
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  verbs:
+  - '*'
+- apiGroups:
+  - ""
+  resources:
+  - nodes/status
+  verbs:
+  - patch
+- apiGroups:
+  - ""
+  resources:
+  - services
+  verbs:
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - services/status
+  verbs:
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - serviceaccounts
+  verbs:
+  - create
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - persistentvolumes
+  verbs:
+  - get
+  - list
+  - update
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - endpoints
+  verbs:
+  - create
+  - get
+  - list
+  - watch
+  - update
+- apiGroups:
+  - coordination.k8s.io
+  resources:
+  - leases
+  verbs:
+  - create
+  - get
+  - list
+  - watch
+  - update
+- apiGroups:
+  - ""
+  resources:
+  - serviceaccounts/token
+  verbs:
+  - create
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: system:cloud-controller-manager
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:cloud-controller-manager
+subjects:
+  - apiGroup: ""
+    kind: ServiceAccount
+    name: cloud-controller-manager
+    namespace: kube-system
+```
+
+After a moment, you should see the cloud controller manager pod was
+successfully deployed.
+
+```bash
+NAME                                 READY   STATUS    RESTARTS        AGE
+aws-cloud-controller-manager-ndbtq   1/1     Running   1 (3h51m ago)   9h
+```
+
+## Deploy the EBS CSI Driver
+
+Now that the cloud controller manager is deployed and can communicate with AWS,
+you are ready to deploy the EBS CSI driver. The easiest way to deploy the
+driver is with the Helm chart. Luckily, Canonical K8s has a built-in helm
+command.
+
+If you want to create encrypted drives, you need to add the statement to the
+policy you are using for the instance.
+
+```json
+{
+  "Effect": "Allow",
+  "Action": [
+      "kms:Decrypt",
+      "kms:GenerateDataKeyWithoutPlaintext",
+      "kms:CreateGrant"
+  ],
+  "Resource": "*"
+}
+```
+
+Then, add the helm repo for the EBS CSI Driver.
+
+```bash
+sudo k8s helm repo add aws-ebs-csi-driver https://kubernetes-sigs.github.io/aws-ebs-csi-driver
+sudo k8s helm repo update
+```
+
+Finally, install the Helm chart, making sure to set the correct region as an
+argument.
+
+```bash
+sudo k8s helm upgrade --install aws-ebs-csi-driver \
+    --namespace kube-system \
+    aws-ebs-csi-driver/aws-ebs-csi-driver \
+    --set controller.region=us-east-2
```

Comment:
Maybe we can put region as a template here like `<REGION>`

Reply:
Sure